index.html

```diff
@@ -440,129 +498,184 @@
 
           callback RemotePlaybackAvailabilityCallback = void(boolean available);
         </pre>
+        <p>
+          A <dfn>RemotePlayback</dfn> object represents a <a>remote playback
+          device</a>.
```

Comment:
I'd rather say something like: A RemotePlayback object is an interface allowing the page to detect, establish connection to and control playback on remote playback devices.